keps/sig-apps/2255-pod-cost/20210112-delete-priority-annotations.yaml

```diff
@@ -0,0 +1,47 @@
+title: Add delete priority annotation for ReplicaSet
```

Comment:
change the file name to kep.yaml, we also need to squash the commits, this is the standard practice.

Reply:
done

Reply:
Does it really make sense to squash now?

Reply:
It went through lots of changes already, so anyone reviewing this will likely have to go through the whole KEP again. We can wait for Janet since she will be the one approving the PR, but my thought was to get the PR in a mergeable state in case you don't have time to make updates anymore.

Reply:
also, do you want to change the KEP from mentioning delete priority to pod cost everywhere (including the KEP and PR title)?

Reply:
The proposal LGTM. +1 to update delete priority to pod cost and add reviewers/approvers. Feel free to squash afterwards.

Reply:
Thank you. I thought that there is much more work to do...
I will go through the KEP again and replace the delete prio to pod cost and add reviewers/approvers and squash afterwards.

Reply:
done